.codespellrc

```diff
@@ -1,5 +1,6 @@
 [codespell]
-skip = .git,*.pdf,*.svg
+skip = .git, *.pdf, *.svg
```

Comment:
Add `*.ipynb` ?

Reply:
I understood to remove them from `.codespellrc`, thank you for the clarification. Notebooks have been exclude from codespell.

Reply:
`.codespellrc` config file is now working correctly and tested.